helm/druid/templates/broker/roleBinding.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.rbac.create }}
```

Comment:
Same question about checking enabled for the druid service, eg: `.Values.broker.enabled`

Reply:
Fixed